message_ix/model/MESSAGE/sets_maps_def.gms

```diff
@@ -202,11 +202,9 @@ Alias(mode, mode2);
 *    * - type_tec_land (type_tec) [#type_tec_land]_
 *      - :math:`\widehat{t} \in \widehat{T}^{LAND} \subseteq \widehat{T}`
 *      - Mapping set of technology types and land use
-*    * - commodity_surplus (commodity)
-*      - :math:`c \in C^{FULL} \subseteq C`
-*      - Commodities related to `Equation COMMODITY_BALANCE_GT`_
-*    * - commodity_shortage (commodity)
+*    * - balance_equality (commodity,level)
```

Comment:
please add description as with the other mapping sets in that table

Reply:
the description is below, do you mean something else